Project.toml

```diff
@@ -18,6 +21,7 @@ DelayEmbeddings = "1.10"
 Distances = "0.9, 0.10"
 NearestNeighbors = "0.4"
 SpecialFunctions = "0.10"
+GroupSlices = "0.0.3"
```

Comment:
Where is this package used? Can it *not* be used? It's just that adding a dependency that is not even in version 0.1.0 seems a really risky thing to do...

Reply:
GroupSlices.jl provides a few functions that replicate the behaviour of matlab's `unique` function (does some extras compared to the julia version). The extra information computed by the matlab version, and by GroupSlices, is used to estimate the transfer operator. 

I guess I could try to rewrite these functions myself for use locally in Entropies.jl, but I figured it was not worth coding up something on my own when functions provided by this package are already very performant. The `groupslices` function and its related functions, are, I guess, very unlikely to change, because they has a singular purpose: do the same as the matlab `unique` function.

Reply:
If the package is MIT licensed you can copy paste them here at no cost. Can you give me a link to the source code of the functions you need?

Reply:
The package seems to be MIT-licensed. The registered version is the fork here: 

https://github.com/mcabbott/GroupSlices.jl/blob/master/src/GroupSlices.jl

I guess we could just define an unexported GroupSlices module in Entropies.jl with the exact code, with a link to the repo.

Reply:
Please copy the code and add license and link to the repo, it feels _really_ unsafe at the moment.

Reply:
Done in 8064c25a04ff80c016c03d971443c77d6f6dbdb9